src/languages/en.js

```diff
@@ -1211,4 +1212,6 @@ export default {
         createAccount: 'Create a new account',
         logIn: 'Log in',
     },
+    allStates: COMMON_CONST.STATES,
+    allCountries: CONST.ALL_COUNTRIES,
```

Comment:
Hmm what's the reason to put `ALL_COUNTRIES` in const instead of here? I think I'd prefer we define it here unless there's something I'm missing

Reply:
OOooooooo well the possible problem there is that we'll only get the correct country ISO if the `countryName` passed is in English, right? Do we need to also make sure this check works for spanish / other langs? 🤔

Reply:
aah true true true, ok i like it! resolving this!